models/transfers/optimism/transfers_optimism_tokens.sql

```diff
@@ -0,0 +1,159 @@
+{{ 
+    config(
+        alias ='tokens', 
+        materialized ='incremental',
+        file_format ='delta',
+        incremental_strategy='merge',
+        unique_key='unique_transfer_id',
+        post_hook='{{ expose_spells(\'["optimism"]\',
+                                    "sector",
+                                    "transfers",
+                                    \'["msilb7", "chuxin"]\') }}'
+    )
+}}
+
+SELECT transfer_from_address,
+        transfer_to_address,
+        contract_address,
+        token_standard,
+        token_type,
+        token_id,
+        transfer_type,
+        -- mint, burn, or transfer
+        CASE
+                WHEN transfer_from_address = '0x0000000000000000000000000000000000000000' THEN 'mint'
+                WHEN transfer_to_address = '0x0000000000000000000000000000000000000000' THEN 'burn'
+                ELSE 'transfer'
+                END 
+        AS transfer_style,
```

Comment:
wonder if `transfer_action` here is better?

Reply:
i like it